src/features/feeds/components/FeedList.tsx

```diff
@@ -30,6 +30,9 @@ export const FeedList = ({
   function handleNetworkSelect(chain: Chain) {
     setSelectedChain(chain.page)
   }
+  useEffect(() => {
+    setSelectedChain("ethereum")
+  }, [])
```

Comment:
Ethereum is initially set as the selected chain to automatically display the Ethereum Chain Price Feed Table upon the initial load.